pytest/tests/mocknet/mirror.py

```diff
@@ -591,6 +594,7 @@ def __call__(self, parser, namespace, values, option_string=None):
     new_test_parser.add_argument('--genesis-protocol-version', type=int)
     new_test_parser.add_argument('--stateless-setup', action='store_true')
     new_test_parser.add_argument('--gcs-state-sync', action='store_true')
+    new_test_parser.add_argument('--epoch-config-overrides', type=json.loads)
```

Comment:
maybe add some help text with an example like in the PR description? Also might be good to warn the user in this help text that this won't work unless the binary has the `--dump-epoch-config` option in `neard init`